### ISSUE TITLE
Release with updated dependencies

### ISSUE THREAD

**Issue by 40thieves** [js-cli/js-liftoff#71]
I'm getting a security vulnerability check from nsp:

```
┌───────────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│               │ Regular Expression Denial of Service                                                                                  │
├───────────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Name          │ minimatch                                                                                                             │
├───────────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Installed     │ 2.0.10                                                                                                                │
├───────────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Vulnerable    │ <=3.0.1                                                                                                               │
├───────────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Patched       │ >=3.0.2                                                                                                               │
├───────────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Path          │ overleaf-editor-storage@0.0.0 > knex@0.11.7 > liftoff@2.2.1 > findup-sync@0.3.0 > glob@5.0.15 > minimatch@2.0.10      │
├───────────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/118                                                                                │
└───────────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

It seems that the findup-sync dependency is already bumped on master, so would it be possible to get a release with this?

**Comment by phated** [comment]
Released as 2.2.3

**Comment by phated** [comment]
Oops, look like we missed 2.2.2 because it was bumped in the package.json instead of using `npm version patch`.  Oh well.
